_posts/en/newsletters/2018-10-09-newsletter.md

```diff
@@ -0,0 +1,283 @@
+hashrate and so is likely to increase mining centralization.[^freq-pow-waste]
+
+Perhaps disproving the saying that "two wrongs don't make a right," this
+talk describes a novel way of combining extension blocks and the
+time-warp attack to allow both upgraded nodes and old nodes to gain the
+same capacity increase and see all the same transactions for validation
+while simultaneously slightly reducing mining centralization risk.
+Upgraded nodes would validate one or more extension blocks providing
+additional block space with a 15 minute average interval, but they'd
+also restrict the time stamps in legacy blocks to ensure a permanent
+(but limited) time warp attack increased the frequency of legacy blocks
+enough to allow them to include the same transactions that previously
+appeared in the extension blocks.
+
+*[video][vid forward blocks], [transcript][tx forward blocks],
+[paper][paper forward blocks], soft fork*
+
+### Compact multi-signatures for smaller blockchains
+
+*Research by Dan Boneh, Manu Drijvers and Gregory Neven*
+
+This talk describes an alternative to the Bellare-Neven-based Schnorr
+signature scheme described in the [MuSig paper][] and [proposed as a
+standard][schnorr pre-bip] for Bitcoin by Pieter Wuille.  The
+alternative method makes use of [pairing-based cryptography][],
+specifically an adaptation of the [Boneh-Lynn-Shacham (BLS) signature
+scheme][bls sigs].  Although pairing-based schemes require an additional
+fundamental security assumption beyond those made by both Bitcoin's
+current ECDSA scheme and proposed Schnorr scheme, the authors present
+evidence that their scheme would produce smaller signatures in general,
+be much more computationally efficient in some multisignature use cases,
+and make it possible to prove which members of the set of signers
+actually worked together to create a threshold signature (i.e.  k-of-m
+signers, e.g. 2-of-3 multisig).
+
+*[video][vid bls msig], [transcript][tx bls msig], [paper
+(pre-print)][paper bls msig], soft fork using segwit script upgrade
+mechanism*
+
+### Accumulators: a scalable drop-in replacement for merkle trees
+
+*Research by Benedikt Bünz, Benjamin Fisch and Dan Boneh*
+
+In Bitcoin and other cryptocurrencies, scalable commitments to sets of
+information---such as transactions or UTXOs---are normally made using
+merkle trees that allow proving an element is a member of the set by
+generating a proof whose size and validation cost is roughly *log2(n)*
+for a set of *n* elements.
+
+This talk describes an alternative method based on RSA accumulators that
+provides many potential benefits: the size of a proof is constant no
+matter how many elements are members of the set, adding or removing
+elements from an accumulator can be efficiently batched (e.g. one update
+per block), and validation that one or more elements are members of a
+set could potentially be much faster (although this has not yet been
```

Comment:
I'm not sure I'd add this last point about being faster. See the section "Too Slow?" in the transcript. Although the proofs are smaller and constant time, the verification time does increase with the number of elements.

My understanding is that the RSA accumulator is still slow compared to hash-based systems, and classgroup performance is completely unknown.